src/FileIndexer.ts

```diff
@@ -65,7 +65,21 @@ export class FileIndexer {
     if (ts.isIdentifier(node) || ts.isStringLiteralLike(node)) {
       const sym = this.getTSSymbolAtLocation(node)
       if (sym) {
-        this.visitSymbolOccurrence(node, sym)
+        /**
```

Comment:
I agree it's not super clear why `this.declarationName(node.parent) === node` identifies a definition. To improve readability, I'd prefer to add a function `private isDefinition(...): boolean` with a docstring than adding an inline comment like this.

Reply:
https://github.com/sourcegraph/scip-typescript/pull/196